<ui version="4.0" >
 <class>frmMarker</class>
 <widget class="QDialog" name="frmMarker" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>549</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>732</width>
    <height>549</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>732</width>
    <height>549</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>MIV Marker Editor</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>190</width>
     <height>471</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
     <horstretch>190</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>190</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>190</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title" >
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout_3" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Select color:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Overlay:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" >
     <widget class="QSlider" name="overlaySlider" >
      <property name="maximum" >
       <number>100</number>
      </property>
      <property name="sliderPosition" >
       <number>50</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" >
     <widget class="QComboBox" name="cboSize" />
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Pointer size in pixels:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QComboBox" name="cboColor" />
    </item>
    <item row="4" column="0" >
     <widget class="QPushButton" name="cmdEraser" >
      <property name="cursor" >
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text" >
       <string>Eraser</string>
      </property>
      <property name="checkable" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="0" >
     <widget class="QComboBox" name="cboMarkerSlice" />
    </item>
    <item row="10" column="0" >
     <widget class="QLabel" name="label_7" >
      <property name="text" >
       <string>Copy marker from:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" >
     <spacer name="verticalSpacer_2" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="14" column="0" >
     <widget class="QPushButton" name="cmdPreview" >
      <property name="text" >
       <string>Segmentation Preview</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" >
     <widget class="QComboBox" name="cboMarkerFrame" />
    </item>
    <item row="15" column="0" >
     <spacer name="verticalSpacer_5" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="0" >
     <widget class="QCheckBox" name="boxInterpolation" >
      <property name="text" >
       <string>Interpolate</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" >
     <spacer name="verticalSpacer_3" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>490</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" >
    <item>
     <widget class="QPushButton" name="cmdSave" >
      <property name="text" >
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cmdCancel" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>70</y>
     <width>511</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <widget class="QVTKWidget" name="vtkWidget" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="MinimumExpanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <widget class="QPushButton" name="cmdUndo" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>Undo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cmdRedo" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>Redo</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cmdRemoveMarker" >
      <property name="text" >
       <string>Remove this marker</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QFrame</extends>
   <header>QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
